### PR TITLE
[Feat: #153058190] Responsive header design

### PR DESCRIPTION
**Pull request by fisayomi**
#### What does this PR do?
- Make the header responsive on mobile devices.

#### How should this be manually tested?
- checkout to branch `ft/responsive-header-design-153058190`
- run `yarn start`

#### Screenshots (if appropriate)

screen shot 2017-11-23 at 5 44 33 pm

screen shot 2017-11-23 at 5 51 55 pm

screen shot 2017-11-23 at 5 52 14 pm

### CODE REVIEW

File: src/common/media.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { css } from 'styled-components';
 export const sizes = {
   desktop: 992,
   tablet: 768,
-  phone: 376
+  phone: 450
```
Comment (oreoluwade) [inline src/common/media.js]: I think 320 by 480 is the common screen resolution for the average mobile device. Unless the `450` here means the maximum width. Could you please confirm this, so we be sure we are designing based on the default webpage sizes for mobile, tablet and desktop devices?

Kindly check [here](https://stackoverflow.com/questions/2364313/is-there-a-standard-webpage-resolution-for-mobile-phones)and other related embedded links you might find in there for clarifications about my concerns.

File: src/components/CustomComponents/Cart.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const ShoppingCart = () => (
+  <GlyphiconContainer to="">
+    <BadgeContainer badgeContent={4} >
+      <ActionAddShoppingCart />
+              Cart
+    </BadgeContainer>
+  </GlyphiconContainer>
+);
+
+export default ShoppingCart;
```
Comment (oreoluwade) [inline src/components/CustomComponents/Cart.js]: Could we make this the last line (or thereabouts of this component/file) so it is immediately obvious at a glance what is being exported?

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -169,8 +189,11 @@ const SearchBar = styled(ToolbarGroup)`
   margin: 20px;
   height: 36px;
```
Comment (oreoluwade) [inline src/components/Header/Header.jsx]: could the `height` and `margin` be in percentages too?

File: src/common/media.js (hunk 1 above)
Comment (fisayomi) [reply src/common/media.js]: Yes, the assumption is that 450 is the maximum width for all phones @oreoluwade

Comment (oreoluwade) [reply src/common/media.js]: Awesome then @fisayomi

File: src/components/Header/Header.jsx (hunk 3 above)
Comment (fisayomi) [reply src/components/Header/Header.jsx]: Percentage seems to be bad here..

File: src/components/CustomComponents/Cart.js (hunk 2 above)
Comment (fisayomi) [reply src/components/CustomComponents/Cart.js]: Done

File: src/components/Header/Header.jsx (hunk 3 above)
Comment (andela-mabdussalam) [reply src/components/Header/Header.jsx]: I am struggling. Can you kindly explain further. @oreoluwade @fisayomi

Comment (fisayomi) [reply src/components/Header/Header.jsx]: What are you struggling with @omorilewa ?

Comment (andela-mabdussalam) [reply src/components/Header/Header.jsx]: Understanding why percentages would be bad in this scenario.

Comment (fisayomi) [reply src/components/Header/Header.jsx]: Using percentage does not make it properly aligned.

Comment (fisayomi) [reply src/components/Header/Header.jsx]: @omorilewa, Is it okay to merge this PR?

Comment (andela-mabdussalam) [reply src/components/Header/Header.jsx]: @fisayomi Yes it is.

**Pull request merged**